### DISCUSSION TITLE
seqr freezes when uploading file in "bulk edit families" tab

### DISCUSSION THREAD

**Discussion by GeorginaAmbriz** [broadinstitute/seqr#3760] [General]
Hello,
I have a local instance of seqr, and I'm using option 2 to annotate and load VCF callsets. I've already uploaded the reference data and the vcf call set (steps 1-4 in option 2).
At the moment, I've opened seqr in the browser with the command:
`xdg-open http://localhost`

In order to add the dataset to my project, I went to my project, I entered to "Edit families and individuals" and I tried to upload the family.tsv file in the "Bulk Edit Family" tab. When I make click in "Click here to upload a table...", seqr freezes. I can't even close the browser. I have to cancel the operation from the terminal. I'm attaching a seqr_logs file.
[seqr_logs.txt](https://github.com/broadinstitute/seqr/files/13506466/seqr_logs.txt)

Any idea of  why this is happening and how to fix it?

Thank you in advance for your support,
Georgina

**Comment by hanars** [comment]
It seems that you tried to upload your data using the "Edit Datasets" button, which is not what that is for and that seems to have gotten seqr into an unhealthy state. Try restarting seqr, and then first you need to create the appropriate families and individuals in seqr without doing anything to edit the datasets. You need to be using the "Bulk Edit Individuals" tab, not the Family tab, as you need to create all the individuals you laded data for in your project. You use the family_id column of that upload to properly create all the families associated with the individuals, you do not do a separate bulk creation of the families. Once you have a record for every individual in your loaded data, you then use the "Edit Datasets" button to link your loaded data to the project, without providing any mapping file
